docs/core/entity.md

```diff
@@ -376,6 +376,85 @@ Called when an entity has their entity_id and hass object assigned, before it is
 
 Called when an entity is about to be removed from Home Assistant. Example use: disconnect from the server or unsubscribe from updates.
 
+## Icons
+
+Every entity in Home Assistant has an icon, which is used as a visual indicator to identify the entity more easily in the frontend. Home Assistant uses the [Material Design Icons](https://materialdesignicons.com/) icon set.
+
+In most cases, Home Assistant will pick an icon automatically based on the entity's domain, `device_class`, and `state`. It is preferred to use the default icon if possible, to provide a consistent experience and to avoid confusion for the user. However, it is possible to override provide a custom icon for an entity.
+
+Regardless of the provided icon, it is always possible for the user to customize the icon to their liking in the frontend.
+
+There are two ways to provide a custom icon for an entity, either by providing icon translations or by providing an icon identifier.
+
+### Icon translations
+
+TThis is the preferred and most modern way to provide a custom icon for an entity. Icon translations work similarly [our regular translations](/docs/internationalization/core#state-of-entities), but instead of translating the state of an entity, they translate the states of an entity to icons.
```

Comment:
```suggestion
This is the preferred and most modern way to provide a custom icon for an entity. Icon translations work similarly [our regular translations](/docs/internationalization/core#state-of-entities), but instead of translating the state of an entity, they translate the states of an entity to icons.
```